pkg/utils/kubeconfig/kubeconfig.go

```diff
@@ -208,11 +208,11 @@ func isValidConfig(p, name string) error {
 }
 
 // MaybeDeleteConfig will delete the auto-generated kubeconfig, if it exists
-func MaybeDeleteConfig(cl *api.ClusterMeta) {
-	p := AutoPath(cl.Name)
+func MaybeDeleteConfig(meta *api.ClusterMeta) {
```

Comment:
I'll split this into another commit, this is just an ancient inconsistency that jumped at me...